### PR TITLE
[ADF-4357] TaskListCloud - change complete task condition

### PR DESCRIPTION
**Pull request by SilviuCPopa**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://issues.alfresco.com/jira/browse/ADF-4357

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.spec.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ describe('TaskHeaderCloudComponent', () => {
 
         fixture.whenStable().then(() => {
             const formNameEl = fixture.debugElement.query(By.css('[data-automation-id="card-textitem-value-assignee"] span'));
-            expect(formNameEl.nativeElement.innerText).toBe('Wilbur Adams');
+            expect(formNameEl.nativeElement.innerText).toBe('superadminuser');
```
Comment (mauriziovitale) [inline lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.spec.ts]: can you use a fake user?

File: lib/process-services-cloud/src/lib/task/task-header/mocks/task-details-cloud.mock.ts [hunk 2]
```diff
@@ -33,7 +33,33 @@ export const taskDetailsCloudMock = new TaskDetailsCloudModel(
...
+        'description': 'This is the description ',
+        'createdDate': 1545048055900,
+        'dueDate': 1545091200000,
+        'claimedDate': null,
+        'priority': 5,
+        'category': null,
+        'processDefinitionId': null,
+        'processInstanceId': null,
+        'status': 'ASSIGNED',
+        'owner': 'hrUser',
```
Comment (mauriziovitale) [inline lib/process-services-cloud/src/lib/task/task-header/mocks/task-details-cloud.mock.ts]: can we have another unit test where the owner is empty?

File: lib/process-services-cloud/src/lib/task/task-header/mocks/task-details-cloud.mock.ts [hunk 3]
```diff
@@ -33,7 +33,33 @@ export const taskDetailsCloudMock = new TaskDetailsCloudModel(
...
+        'parentTaskId': null,
+        'formKey': null,
+        'lastModified': 1545048055900,
+        'lastModifiedTo': null,
+        'lastModifiedFrom': null,
+        'standAlone': true
+    }
+);
+
+export const taskDetailsCloudMock2 = new TaskDetailsCloudModel(
```
Comment (mauriziovitale) [inline lib/process-services-cloud/src/lib/task/task-header/mocks/task-details-cloud.mock.ts]: can we have a meaningful name?

File: lib/process-services-cloud/src/lib/task/task-header/mocks/task-details-cloud.mock.ts (hunk 2 above)
Comment (mauriziovitale) [reply lib/process-services-cloud/src/lib/task/task-header/mocks/task-details-cloud.mock.ts]: add another case when the status is created

File: lib/process-services-cloud/src/lib/task/services/task-cloud.service.ts [hunk 4]
```diff
@@ -71,7 +71,7 @@ export class TaskCloudService {
      */
     canCompleteTask(taskDetails: TaskDetailsCloudModel): boolean {
         const currentUser = this.identityUserService.getCurrentUserInfo().username;
-        return taskDetails.assignee && taskDetails.owner === currentUser && !taskDetails.isCompleted();
+        return taskDetails.assignee && taskDetails.assignee === currentUser && !taskDetails.isCompleted();
```
Comment (mauriziovitale) [inline lib/process-services-cloud/src/lib/task/services/task-cloud.service.ts]: is better to check taskDetails.isAssigned
